frontend/src/routes/index.jsx

```
@@ -0,0 +1,48 @@
+import App from "@/App";
+import { AuthorizedCheck } from "@/components";
+import { About, Dashboard, Error, Home, Profile, SignIn, SignUp, Statistics } from "@/pages";
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)